### ISSUE TITLE
[Bug] 5.0.0.2478 RequestedThemeChanged event not being triggered on Android

### ISSUE THREAD

**Issue by rmachain-bairesdev** [xamarin/Xamarin.Forms#15421]
Description
Hello, I can't get RequestedThemeChanged event to work on Android. On iOS works perfectly. In Android

Steps to Reproduce
Download SystemThemesDemo application
Add following code in contructor of the App class:
Application.Current.RequestedThemeChanged += (s, a) =>
            {
                // Respond to the theme change
                Console.WriteLine("Theme changed");
            };
Run application and change theme to dark/light in settings and it will work sometimes, **but if it is done throughout the top bar shortcut "Dark theme" button it won't be triggering.**

Expected Behavior
Application UI change and RequestedThemeChanged call

**Comment by jfversluis** [comment]
Make sure that you have the right configuration in place as per the documentation: https://docs.microsoft.com/xamarin/xamarin-forms/user-interface/theming/system-theme-changes#react-to-theme-changes

> To respond to theme changes on Android you must include the ConfigChanges.UiMode flag in the Activity attribute of your MainActivity class.

**Comment by jfversluis** [comment]
You mention 5.0.0.2478 specifically. Did it work on earlier version?

**Comment by rmachain-bairesdev** [comment]
Yes was working, I will return to the previous version and test again.

**Comment by rmachain-bairesdev** [comment]
Update:

Hey jfversluis. I tested and it's working well, when it fails is when I tested on an Android Tablet and it is in Landscape mode, if I use the shortcut to change between dark/light mode it doesn't change.
